ecommerce/src/Relecloud.Web/Services/SqlDatabaseConcertRepository/ConcertDataContext.cs

```diff
@@ -46,9 +43,9 @@ public void Initialize()
                 var concertsTableName = nameof(Concerts);
                 // Note: the cast to string is to work around an issue in Entity Framework Core 2.0 with string interpolation.
                 // See https://github.com/aspnet/EntityFrameworkCore/issues/9734.
-                this.Database.ExecuteSqlCommand((string)$"ALTER TABLE [{concertsTableName}] ENABLE CHANGE_TRACKING WITH (TRACK_COLUMNS_UPDATED = ON)");
+                this.Database.ExecuteSqlRaw((string)$"ALTER TABLE [{concertsTableName}] ENABLE CHANGE_TRACKING WITH (TRACK_COLUMNS_UPDATED = ON)");
```

Comment:
I don't know if the cast to string is still required here as a workaround for the (old) EF string interpolation issue mentioned above?